### DISCUSSION TITLE
CSS Battle #32 – Band-aid

### DISCUSSION THREAD

**Discussion by meg-gutshall** [Virtual-Coffee/virtualcoffee.io#1078] [CSS Battles]
### Link to battle:

[Let's battle! ⚔️](https://cssbattle.dev/play/32)

### Copy the code block below to format your comment on the discussion thread:

````md
<details>
<summary>Code Source  – <strong>score {character count}</strong></summary>
<!-- have to be followed by an empty line! -->

```html
your cssbattles code here!
```

</details>
````

### What others will see:

This will result in a nice hidden bit like so:

<details>
<summary>Code Source  – <strong>score {character count}</strong></summary>

```html
your cssbattles code here!
```
</details>

**Comment by meg-gutshall** [comment]
<details>
<summary>First try  – <strong>622.07 {304}</strong></summary>


```html
<div class=y>
  <div class=g></div>
</div>
<style>
  body {
    display: grid;
    place-items: center;
  }
  .y {
    rotate: -45deg;
    background: #F3AC3C;
  }
  .g {
    rotate: 90deg;
    background: #A3A368;
  }
  div {
    width: 50px;
    height: 200px;
    mix-blend-mode: screen;
  }
</style>
```

</details>

<details>
<summary>Refactored & minified  – <strong>674.03 {177}</strong></summary>


```html
<div y><div g><style>body{display:grid;place-items:center;}[y]{rotate:-45deg;background:#F3AC3C}[g]{rotate:90deg;background:#A3A368}div{width:50;height:200;mix-blend-mode:screen
```

</details>

<details>
<summary>Refactored some more after @Narigo goaded me  – <strong>679.14 {170}</strong></summary>


```html
<div><div g><style>body{display:grid;place-items:center}[g]{rotate:90deg;background:#A3A368}div{rotate:-45deg;background:#F3AC3C;width:50;height:200;mix-blend-mode:screen
```

</details>

**Comment by Narigo** [comment]
<details>
<summary>Smaller than @meg-gutshall again 😅 – <strong>685.41 {162}</strong></summary>


```html
<r><r><style>*+*,r{display:grid;place-items:center}r{rotate:45deg;width:50;height:200;background:#A3A368;mix-blend-mode:screen}r>r{rotate:90deg;background:#F3AC3C
```

</details>

**Reply by aishwarya-mali** [reply]
Are tags of any character allowed? I saw you use a, b, c in other challenge?

**Comment by aishwarya-mali** [comment]
<details>
<summary>Mix blend mode – <strong>655.62{207}</strong></summary>


```html
<p><style>*{margin:0}p{margin:50 175}p:after,p:before{content:'';height:200;width:50;position:absolute}p:before{background:#a3a368;rotate:45deg}p:after{background:#f3ac3c;rotate:-45deg;mix-blend-mode:screen}
```

</details>
